Microsoft.Toolkit.Uwp.UI.Controls/ColumnResizer/ColumnResizer.Events.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// ColumnResizer is a UI control that add the resizing functionality to a Grid Column.
+    /// </summary>
+    public partial class ColumnResizer
+    {
+        /// <summary>
+        /// Parent Grid On Pointer Moved Event
+        /// </summary>
+        private void Resizable_PointerMoved(object sender, PointerRoutedEventArgs e)
+        {
+            if (_isResizing)
+            {
+                var point = e.GetCurrentPoint(Resizable);
+                if (point.Position.X - FixedOffset >= 0)
+                {
+                    CurrentColumn.Width = new GridLength(point.Position.X - FixedOffset);
```

Comment:
Have you tried this with multple columns? Looking at the math, it seems to me the CurrentColumn.Width is set to the distance from the left side of the parent grid to the mouse cursor, but if there's more than one column to the left of this column resizer instance, that wouldn't work, since the previous columns' widths should be subtracted.